cmd/node-termination-handler.go

```diff
@@ -223,7 +222,7 @@ func main() {
 			QueueURL:                      nthConfig.QueueURL,
 			InterruptionChan:              interruptionChan,
 			CancelChan:                    cancelChan,
-			SQS:                           sqs.New(sess),
+			SQS:                           sqsevent.GetSqsClient(sess),
 			ASG:                           autoscaling.New(sess),
```

Comment:
should we also add retry for asg client?

Reply:
We could keep use default retryer for asg client.